lib/src/Uncertainty/Distribution/openturns/PointConditionalDistribution.hxx

```diff
@@ -97,6 +97,12 @@ public:
   Scalar computeConditionalQuantile(const Scalar q, const Point & y) const override;
```

Comment:
According to Codecov the following methids are untested:
- computeSequentialConditionalPDF 
- computeSequentialConditionalCDF 
- computeConditionalQuantile
- computeSequentialConditionalQuantile
- getIsoProbabilisticTransformation
- getInverseIsoProbabilisticTransformation